README.md

```diff
@@ -205,31 +180,23 @@ pm2 start current/pm2.config.js --env production
-│                                     ││ results from LIMS sample endpoint.                                                      │
-│                                     ││ IPR-API-syncWorker > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:21][INFO] Starting to   │
-│                                     ││ process sample results.                                                                 │
-│                                     ││ IPR-API-syncWorker > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:21][INFO] Resulting     │
-│                                     ││ in 7 disease libraries that have pathology information and need additional library      │
+│[101] IPR-API        Mem:  88 MB     ││ IPR-API > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:20][INFO] Retrieved assembly       │
+│                                     ││ results                                                                                 │
+│                                     ││ IPR-API > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:20][INFO] Querying DB for all      │
+│                                     ││ tracking tasks with symlinks pending                                                    │
+│                                     ││ IPR-API > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:20][INFO] Tasks requiring symlink  │
+│                                     ││ lookup 17                                                                               │
+│                                     ││ IPR-API > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:20][INFO] Starting query for       │
+│                                     ││ retrieving library lane targets for 51 libraries                                        │
+│                                     ││ IPR-API > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:20][INFO] Target lanes determined  │
+│                                     ││ for each library. Querying for aligned libcores                                         │
+│                                     ││ IPR-API > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:21][INFO] Checked in symlinks for  │
+│                                     ││ 0 tasks.                                                                                │
+│                                     ││ IPR-API > 2018-01-05 14:22 -08:00: [2018-01-05 14:22:21][INFO] Resulting in 7 disease   │
+│                                     ││ libraries that have pathology information and need additional library                   │
 │                                     ││ details to differentiate RNA vs DNA.                                                    │
 └─────────────────────────────────────┘└─────────────────────────────────────────────────────────────────────────────────────────┘
-┌─ Custom metrics (http://bit.ly/code─┐┌─ Metadata ──────────────────────────────────────────────────────────────────────────────┐
```

Comment:
Is anyone else seeing this link to a wikipedia article on Irish bread?? Weird

Reply:
Haha, yeah I saw that. I'm pretty sure at some point the url got cutoff and is suppose to be http://bit.ly/code-metrics as shown on their website here https://pm2.keymetrics.io/docs/usage/process-management under the 'Terminal Dashboard' section.